src/services/atomic-swap.service.js

```diff
@@ -74,6 +74,26 @@ const serviceCreate = async (walletId, partialTx, password) => {
   return { proposalId: id };
 };
 
+/**
+ * Creates the proposal on the Atomic Swap Service, handling errors that may occur on the process
+ * @param updateParams Parameters related to the proposal itself
+ * @param {string} updateParams.partialTx Serialized PartialTx
+ * @param {string} updateParams.proposalId Proposal identifier
+ * @param {string} updateParams.password Proposal password on the Atomic Swap Service
+ * @param {number} updateParams.version Proposal version on the Atomic Swap Service
+ * @param {string} [updateParams.signatures] Proposal signatures, if any
+ * @returns {Promise<{ success: boolean }>} Returns if the operation was successful
+ */
+const serviceUpdate = async updateParams => {
+  const { success } = await swapService.update(updateParams);
+
+  if (!success) {
+    throw new Error('Unable to update the proposal on the Atomic Swap Service');
+  }
+
+  return { success };
```

Comment:
Should we return the success? It will always be true due to the check above, right?

Reply:
Improved to returning `Promise<void>` on [4dbf6cd](https://github.com/HathorNetwork/hathor-wallet-headless/pull/286/commits/4dbf6cd55b6b7dd5943ae77654fb6ef1fc952721)